packages/@aws-cdk/aws-lambda-golang/README.md

```diff
@@ -0,0 +1,246 @@
+# Amazon Lambda Golang Library
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![cdk-constructs: Experimental](https://img.shields.io/badge/cdk--constructs-experimental-important.svg?style=for-the-badge)
+
+> The APIs of higher level constructs in this module are experimental and under active development.
+> They are subject to non-backward compatible changes or removal in any future version. These are
+> not subject to the [Semantic Versioning](https://semver.org/) model and breaking changes will be
+> announced in the release notes. This means that while you may use them, you may need to update
+> your source code when upgrading to a newer version of this package.
+
+---
+
+<!--END STABILITY BANNER-->
+
+This library provides constructs for Golang Lambda functions.
+
+To use this module you will need to have Docker installed.
```

Comment:
Is it possible to offer a local bundling experience that does not require docker to be installed? (similar to how `NodejsFunction` allows you to install `esbuild` locally)

Reply:
Good catch, this comment should be removed since it does support local bundling.

Reply:
Updated!